### PR TITLE
add travis configuration

### PR DESCRIPTION
**Pull request by adityaprakash-bobby**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: ruby
+cache: bundler # caching to speed up dependency installation
+rvm:
+- 2.4.1
```
Comment (adityaprakash-bobby) [inline .travis.yml:4]: Replacing with the correct version of ruby helps.

File: script/build [hunk 2]
```diff
@@ -0,0 +1 @@
+bundle exec jekyll build
```
Comment (adityaprakash-bobby) [inline script/build:1]: Good to go.

File: script/testlinks [hunk 3]
```diff
@@ -0,0 +1,2 @@
+bundle exec jekyll build
+bundle exec htmlproofer ./_site --check-html
```
Comment (adityaprakash-bobby) [inline script/testlinks:2]: Good to go.

**Review by adityaprakash-bobby — COMMENTED** [review]
LGTM

**Review by kiminohero — APPROVED** [review]

**Pull request merged**